crowdsec-docs/docs/intro.mdx

```diff
@@ -26,8 +34,9 @@ The crowd-sourced aspect allows sharing attack information among users, enhancin
 In addition to the core "detect and react" mechanism, CrowdSec is committed to several other key aspects:
 
 - **Easy Installation**: Effortless out-of-the-box installation on all [supported platforms](/getting_started/versions_matrix.md).
```

Comment:
we could mention the word package or packaged in there ?
And add mass compatibility

- **Mass compatibility**: Coded in GOlang, it can run on virtually any system
- **Effortless** out-of-the-box installation and packaged on [many supported platforms](/getting_started/versions_matrix.md).

Reply:
I think this `Mass compatibility: Coded in GOlang, it can run on virtually any system` is implied and expected